src/Auth/Common.elm

```diff
@@ -230,3 +252,7 @@ type alias SessionId =
 
 type alias ClientId =
     String
+
+
+isDev =
+    Env.mode == Env.Development
```

Comment:
Will need to revert this as `Env.*` is a project-code concept and won't be available in the package-code context.